RecoJets/FFTJetProducers/src/FFTJetParameterParser.cc

```diff
@@ -387,23 +389,23 @@ namespace fftjetcms {
     return return_type(nullptr);
   }
 
-  std::unique_ptr<fftjet::ClusteringTreeSparsifier<fftjet::Peak, long> > fftjet_ClusteringTreeSparsifier_parser(
+  std::unique_ptr<fftjet::ClusteringTreeSparsifier<fftjet::Peak, long>> fftjet_ClusteringTreeSparsifier_parser(
       const edm::ParameterSet& ps) {
-    typedef std::unique_ptr<fftjet::ClusteringTreeSparsifier<fftjet::Peak, long> > return_type;
+    typedef std::unique_ptr<fftjet::ClusteringTreeSparsifier<fftjet::Peak, long>> return_type;
```

Comment:
This is now not needed any more inside this method.

Reply:
removing it